cmd/loom/loom.go

```diff
@@ -639,18 +640,12 @@ func loadAppStore(cfg *config.Config, logger *loom.Logger, targetVersion int64)
 		}
 	} else if cfg.AppStore.Version == 3 {
 		logger.Info("Loading Multi-Writer App Store")
-		iavlStore, err := store.NewIAVLStore(db, cfg.AppStore.MaxVersions, targetVersion, cfg.AppStore.IAVLFlushInterval)
-		if err != nil {
-			return nil, err
-		}
-		evmStore, err := loadEvmStore(cfg, iavlStore.Version())
-		if err != nil {
-			return nil, err
-		}
-		appStore, err = store.NewMultiWriterAppStore(iavlStore, evmStore, cfg.AppStore.SaveEVMStateToIAVL)
+		appStore, err = store.LoadMultiWriterAppStore(cfg.AppStore, cfg.EvmStore, db, cfg.AppStore.MaxVersions,
+			cfg.Metrics.Database, cfg.RootPath())
```

Comment:
Format this like so:
```golang
appStore, err = store.LoadMultiWriterAppStore(
    cfg.AppStore, cfg.EvmStore, db, cfg.AppStore.MaxVersions,
    cfg.Metrics.Database, cfg.RootPath(),
)
```
Also the fourth argument is supposed to be the `targetVersion`, doesn't make sense to pass in `cfg.AppStore.MaxVersions` there.